drivers/core_demographics_report/app/models/core_demographics_report/high_acuity_calculations.rb

```diff
@@ -39,60 +39,134 @@ def high_acuity_data_for_export(rows)
       rows['_High Acuity Type'] ||= []
       rows['*High Acuity Type'] ||= []
       rows['*High Acuity Type'] += ['High Acuity Type', nil, 'Count', 'Percentage', nil]
+      available_coc_codes.each do |coc_code|
+        rows['*High Acuity Type'] += ["#{coc_code} Client"]
+        rows['*High Acuity Type'] += ["#{coc_code} Percentage"]
+      end
+      rows['*High Acuity Type'] += [nil]
       available_high_acuity_types.invert.each do |id, title|
         rows["_High Acuity Type_data_#{title}"] ||= []
         rows["_High Acuity Type_data_#{title}"] += [
           title,
           nil,
           high_acuity_count(id),
           high_acuity_percentage(id) / 100,
+          nil,
         ]
+        available_coc_codes.each do |coc_code|
+          rows["_High Acuity Type_data_#{title}"] += [
+            high_acuity_count(id, coc_code.to_sym),
+            high_acuity_percentage(id, coc_code.to_sym) / 100,
+          ]
+        end
       end
       rows
     end
 
-    private def high_acuity_client_ids(key)
-      high_acuity_clients[key]
+    private def high_acuity_client_ids(key, coc_code = base_count_sym)
+      high_acuity_clients[key][coc_code]
+    end
+
+    private def hoh_client_ids
+      @hoh_client_ids ||= hoh_scope.pluck(:client_id)
     end
 
     def available_high_acuity_types
       {
         'Client' => :client,
         'Household' => :household,
+        'Adult only Households' => :without_children,
+        'Adult and Child Households' => :with_children,
+        'Child only Households' => :only_children,
+        'Youth Only' => :unaccompanied_youth,
+        '1 Disability' => :one_disability,
       }
     end
 
+    private def without_children
+      @without_children ||= enrollment_ids_in_household_type(:without_children)
+    end
```

Comment:
I think having these in multiple concerns is unnecessary, and potentially dangerous if they have different definitions.  I't probably worth gathering them up in the `report_concern.rb` or a `household_type_definitions` concern and then including that in the report before these concerns.  If there are differences in the implementation, then we should make sure they have different names.  
This is one of the pitfalls of these reports with lots of concerns, it can be hard to track down method replacement and keep track of the API.